workshops/personal_website/README.md

```diff
@@ -0,0 +1,239 @@
+
+Here is the final product you will end up with:
+
+![](http://placehold.it/200?text=Preview)
+
+> Open the [live demo](#TODO).
+>
+> View [the code](#TODO).
+
+## Part I: Setup Your Tools
+
+### GitHub
+
+- What is GitHub?
+- What are we using GitHub for?
+- Let's setup your GitHub account.
+- Write down your email, username, and password somewhere you won't forget
+  (you'll need it later in this tutorial and later in the semester).
+
+### Cloud9
+
+- What is Cloud9?
+- What are we using Cloud9 for?
+- Let's create a Cloud9 Account.
+- Let's create and setup a Cloud9 workspace.
+- Write down your email, and password somewhere you won't forget.
+  (you'll be using it for the rest of the semester).
+
+## Part II: HTML
+
+- What is HTML?
+- What are we using HTML for?
+
+### Step 1: Create a folder for your personal website
+
+- Let's create a website
+
+### Step 2: Creating your HTML file
+
+### Step 3: Add the standard HTML template
+
+- Let's paste the below code
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+
+</body>
+</html>
+```
+
+### Step 4: Add Content To The Body
+
+- Let's type the following.
+- Mention copying and pasting vs re-typing
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  I am Drake
+  All your hotlines belong to me.
+</body>
+</html>
+```
+
+### Step 5: Formatting the Heading
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1>I am Drake</h1>
+  All your hotlines belong to me.
+</body>
+</html>
+```
+
+### Step 6: Formatting the Paragraph
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+### Step 7: Adding an Image
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+Our image is too big. CSS to the rescue.
+
+## Part III: CSS
+
+- What is CSS?
+- What are we going to use it for?
+  - Show before and after CSS
+
+### Creating the CSS file
+
+### Linking the CSS in the HTML
+
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <img src="https://media.giphy.com/media/vsyKKf1t22nmw/giphy.gif">
+  <h1>I am Drake</h1>
+  <p>All your hotlines belong to me.</p>
+</body>
+</html>
+```
+
+### Resizing the Image
+
+```css
+img {
+    width: 200px;
+}
+```
+
+### Centering the Body
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+}
+```
+
+### Setting the Background Color
+
+```css
+img {
+    width: 200px;
+}
+
+body {
+    text-align: center;
+    background-color: #34495e;
+}
+```
+
+Use http://colorhunt.co to choose your favorite colors.
+
+<!-- I specifically like colorhunt.co because it only displays the hex colors and the user cannot copy and paste the color values so must retype them-->
```

Comment:
Hmm. So I'm imagining the scenario where someone makes a pull request to change colorhunt.com to say flatuicolors.com because in flatuicolors.com, you don't have to retype the color values. If the comment was there, they would understand why I made the decision to use colorhunt.com over flatuicolors.com

Do you think conveying this context is valuable @zachlatta?